libtesting-ui/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion androidVersions.compileSdkVersion
+    buildToolsVersion androidVersions.buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion androidVersions.minSdkVersion
+        targetSdkVersion androidVersions.targetSdkVersion
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'consumer-rules.pro'
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    api project(':libtesting-utils')
+    // downgrade because of https://github.com/mockk/mockk/issues/281
+    api 'io.mockk:mockk-android:1.9.2'
```

Comment:
Why does the instrumentation testing lib needs mocks?

Reply:
We still may want to mock something  (i.e. our own dependencies) inside instrumentation tests. Ofc we don't use it for now, so we may add it later